_studio/shared/umc/codec/jpeg_common/src/bitstreamin.cpp

```diff
@@ -65,8 +65,9 @@ JERRCODE CBitStreamInput::Detach(void)
     m_pData = 0;
   }
 
-  m_in         = 0;
-  m_pData      = 0;
+  delete m_in;
+  m_in = nullptr;
```

Comment:
hm. We did not have `delete m_in` before. Why? We had a memory leak? Can you, please, provide valgrind test results for the command line you are using to verify that change don't break anything. I hope to see sample decode or encode command line:
`valgrind --tool=memcheck --leak-check=full --log-file=vlg.log <cmdline>`

I am looking for the last section with the memleaks only (omit errors). Please, provide 3 results for each commit you have.

Reply:
no leaks before: m_in - pointer to local variable (on stack) which will be deleted when leave function's body